spec/jobs/clock/subscriptions_to_be_terminated_job_spec.rb

```diff
@@ -63,23 +84,29 @@
       end
     end
 
-    context 'with customer timezone' do
-      let(:ending_at) { DateTime.parse('2022-10-21 00:30:00') }
-
-      before do
-        subscription1.customer.update!(timezone: 'America/New_York')
+    context 'when the same alert webhook has already been triggered 30 days ago' do
+      let(:webhook_alert1) do
+        create(
+          :webhook,
+          :succeeded,
+          object_id: subscription1.id,
+          webhook_type: 'subscription.termination_alert',
+          created_at: ending_at - 45.days,
+        )
       end
 
-      it 'takes timezone into account' do
+      before { webhook_alert1 }
+
+      it 'sends webhook that subscription is going to be terminated for the right subscriptions' do
         current_date = ending_at - 15.days
 
         travel_to(current_date) do
           expect do
             described_class.perform_now
           end
             .to have_enqueued_job(SendWebhookJob)
-            .with('subscription.termination_alert', Subscription)
-            .exactly(0).times
+              .with('subscription.termination_alert', Subscription)
```

Comment:
Layout/MultilineMethodCallIndentation: Use 2 (not 4) spaces for indenting an expression spanning multiple lines.